website/docs/core/brands.md

```diff
@@ -3,7 +3,7 @@ title: Brands
 slug: /brands
 ---
 
-authentik support soft multi-tenancy. This means that you can configure several options depending on domain, but all the objects like applications, providers, etc, are still global. This can be handy to use the same authentik instance, but branded differently for different domains.
+You can configure several differently "branded" options depending on the associated domain, even though objects such as applications, providers, etc, are still global. This can be handy to use the same authentik instance, but branded differently for different domains.
```

Comment:
Suggestion: "This makes it handy to use a single authentik instance to provide different visible brands at different domains."